spec/classes/mycnf_template_spec.rb

```diff
@@ -36,19 +36,24 @@
 
         describe 'ssl set to true' do
           let(:params) {{ :override_options => { 'mysqld' => { 'ssl' => true }}}}
-          it { is_expected.to contain_file('mysql-config-file').with_content(/ssl/) }
+          it { is_expected.to contain_file('mysql-config-file').with_content(/^ssl$/) }
           it { is_expected.to contain_file('mysql-config-file').without_content(/ssl = true/) }
         end
 
         describe 'ssl set to false' do
           let(:params) {{ :override_options => { 'mysqld' => { 'ssl' => false }}}}
-          it { is_expected.to contain_file('mysql-config-file').with_content(/ssl = false/) }
+          it { is_expected.to contain_file('mysql-config-file').without_content(/^ssl$/) }
```

Comment:
the `^anchoring$` here makes me suspicious of false negatives? postitives?
wrong non-matches anyway…